src/components/ec-submenu/ec-submenu.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div
+    ref="submenu"
+    v-if="submenu"
+    class="ec-submenu"
+  >
+    <div
+      ref="header"
+      class="ec-submenu__header-container"
+    >
+      <ul
+        class="ec-submenu__header"
+        size="24"
+      >
+        <li
+          v-for="(menuItem, index) in submenu"
+          :key="menuItem.index"
+          class="ec-submenu__header-item"
+          :class="{'ec-submenu__header-item--active': index === activeIndex}"
+          @click="$emit('change', index)"
+        >
+          <a
+            v-if="menuItem.href"
+            :href="menuItem.href"
+            class="ec-submenu__header-title"
+            @click.prevent
+          >
+            {{ menuItem.headerTitle }} + {{ menuItem.additionalText }}
+          </a>
+
+          <router-link
+            v-if="menuItem.route"
+            :to="menuItem.route"
+            class="ec-submenu__header-title"
+            @click.prevent
+          >
+            {{ menuItem.headerTitle }} + {{ menuItem.additionalText }}
+          </router-link>
+        </li>
+      </ul>
+    </div>
+
+    <main class="ec-submenu__main">
+      <transition-group
+        name="ec-submenu__fade"
+        tag="div"
+      >
+        <div
+          v-for="(menuItem, index) in submenu"
+          :key="index"
+          v-show=" index === activeIndex"
+          :class="{isActive: index === activeIndex}"
+          class="fade-item"
+        >
+          <slot
+            :name="menuItem.slotName"
+          />
+        </div>
+      </transition-group>
+    </main>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EcSubmenu',
+  model: {
+    prop: 'activeIndex',
+    event: 'change',
+  },
+  props: {
+    submenu: {
+      type: Array,
+      required: true,
+    },
+    activeIndex: {
+      type: Number,
+      default: 0,
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+@import '../../scss/settings/index';
+@import '../../scss/tools/index';
+
+$ec-submenu-body: $level-3-body-and-headings !default;
+$ec-submenu-hover: $level-4-tech-blue !default;
+$ec-submenu-disabled: $level-6-disabled-lines !default;
+$ec-submenu-white: $white !default;
+
+.ec-submenu {
+  position: relative;
+  z-index: 0;
+
+  &__header {
+    min-width: 100px;
+    margin-bottom: 12px;
+    white-space: nowrap;
+    display: flex;
+    flex-direction: column;
+
+    @include media__from-1024 {
+      flex-direction: row;
+      border-bottom: 1px solid $ec-submenu-disabled;
+    }
+  }
+
+  &__header-item {
+    bottom: -1px;
+    cursor: pointer;
+    display: inline-block;
+    padding: 8px 16px 4px 16px;
+    position: relative;
+    text-align: center;
+  }
+
+  &__header-title {
+    text-decoration: none;
+    outline: none;
+    color: $ec-submenu-body;
+    transition: color 0.3s ease-out;
+
+    &:focus {
+      outline-width: 0;
```

Comment:
I think we should change the color when gaining focus. Tabbing into submenu has no visual indication.

Reply:
Changed it to the same kind of blue for focus. The user can still understand what is active tab because with have the line underneath. I also confirmed this with Uve.